quizzes/midterm.md

```diff
@@ -1,19 +1,20 @@
 1. Circle all 5 of the problems with the code below.
 
 	```ruby
-	class Monkey < Monkey
+	class Monkey < Monkey  > 1this would not work 
 	  def init(my_vars)
-	    my_vars = my_vars
+	    my_vars = my_vars		>2should be	@my_vars = my_vars
 	  end
 
 	  private
 	  def eat_banana
-	    "Om Om Om, oooh aaaah aaaah"
+	    "Om Om Om, oooh aaaah aaaah"    > 3can't call outside class 
 	  end
 	end
 
 	monkey = Monkey.new
-	monkey.eat_banana
+	monkey.eat_banana  	>4can't call outside class with out it is in a private method need a seperate method first to call it within the class >5 should also be Monkey.eat_banana
+	>need another method
 	```
 
 1. What is returned from the statement below?
```

Comment:
0/10. This is a tricky question. The array gets initialized with all the values.